### PR TITLE
construct MARC record from C0140 ASpace component

### PR DESCRIPTION
**Pull request by regineheberlein**

### CODE REVIEW

**Comment by regineheberlein** [comment]
Note: the latest revert is due to some last-minute strategizing with the archivists that's now going to be approached via an indexing proposal

File: reports/C0140/get_C0140_ao_data.rb [hunk 1]
```diff
@@ -0,0 +1,413 @@
+# frozen_string_literal: true
+
+require 'archivesspace/client'
+require 'json'
+require 'nokogiri'
+require_relative '../../helper_methods'
+
+def remove_tags(text)
```
Comment (sandbergja) [inline reports/C0140/get_C0140_ao_data.rb:8]: This method seems opaque to me.  Could you add a comment showing a typical input and output?

**Review by sandbergja — CHANGES_REQUESTED** [review]
I got confused right away! 🤣  It looks good to me, @regineheberlein -- just wanted some clarification about one of the methods. :-)

File: reports/C0140/get_C0140_ao_data.rb (hunk 1 above)
Comment (regineheberlein) [reply reports/C0140/get_C0140_ao_data.rb:8]: Ah, I remember! So for reasons to do with pulfalight display, some of the ASpace note fields contain EAD markup (like `<p>` tags and `<chronlist>` for example). The EAD markup breaks the marcXML export, so this method removes them. I'll push a comment.

**Review by sandbergja — APPROVED** [review]
Looks good, and thanks!  Congratulations on finishing this project, @regineheberlein ! 🎉🎉🎉🎉

**Pull request merged**